src/index.html

```diff
@@ -5,8 +5,402 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header">
+      <a href="#home">
+        <img
+          src="./images/logo.png"
+          alt="moyo"
+          class="Image"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">APPLE</a>
+          </li>
+          <li class="nav__item">
```

Comment:
```suggestion
          </li>
          
          <li class="nav__item">
```
[poin 2] in (https://github.com/mate-academy/layout_catalog/blob/master/checklist.md)